components/Filter/utils.js

```diff
@@ -20,14 +20,17 @@ export const getUniqueFieldValues = (issues) => {
 }
 
 // Filter issues based on selected filters
-export const filterIssuesBySelectedFilters = (issues, selectedFilters) => {
+export const filterIssuesBySelectedFilters = (issues, selectedFilters, searchQuery) => {
   return issues.filter((issue) =>
     Object.entries(selectedFilters).every(
       ([filterName, selectedValue]) =>
         !selectedValue ||
         issue.fields.some((field) => field.field === filterName && field.value === selectedValue)
+    ) &&
```

Comment:
this const has a lot of responsibility, can you separate the responsibilities into more consts?

Reply:
@Guboehm and @nomadbitcoin, I made an adjustment to this component because before the search was working but the filter had an error and was not filtering. Despite using the same "const", I separated the two functions into issuesFilter and issuesSearch. I chose this way because the search also has to work within an already established filter.